onyo/lib/onyo.py

```diff
@@ -510,6 +512,32 @@ def _fsck_clean_tree(self) -> bool:
 
         return True
 
+    def _fsck_no_pseudo_keys(self) -> bool:
+        """
+        Check that no asset contains any pseudo-key names.
+        """
+        assets_with_pseudo_keys = {}
+        pseudo_keys = ["type", "make", "model", "serial"]
+
+        for asset in self.assets:
+            contents = self._read_asset(asset)
+
+            for p in pseudo_keys:
+                if p in contents:
+                    if asset not in assets_with_pseudo_keys.keys():
+                        assets_with_pseudo_keys[asset] = []
+                    assets_with_pseudo_keys[asset].append(p)
+
+        if assets_with_pseudo_keys:
+            log.error('The following assets contain pseudo keys:\n' +
```

Comment:
This should be even more explicit. The user may not know what a pseudo key is, or that it is not allowed. Currently this message just states a fact. At minimum it should add "which are reserved for use in the asset filename."